glide.yaml

```diff
@@ -1,3 +1,8 @@
 package: github.com/wantedly/developers-account-mapper
 import:
 - package: github.com/mitchellh/cli
+- package: github.com/aws/aws-sdk-go
+  subpackages:
+  - aws
+  - aws/session
+  - aws/service/dynamodb
```

Comment:
To keep every build reproducible, please commit `glide.lock` in this repository.